### PR TITLE
Fix typo in README ( there -> they )

### PR DESCRIPTION
**Pull request by bmfs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ I know it's very common, but it is.
 ## Design Philosophy
 
 It depends on your purpose of your SDK service and usage,
```
Comment (huei90) [inline README.md]: Can you please also update this line

```diff
- It depends on your purpose of your SDK service and usage,
+ It depends on the purpose of your SDK service and usage,
```

**Comment by huei90** [comment]
Thanks @bmfs for updating the error typo.

Better if you can help on fluent the sentences 🍻

File: README.md (hunk 1 above)
Comment (bmfs) [inline README.md]: Done

**Comment by huei90** [comment]
Super Thanks!!!!!

**Pull request merged**